### PR TITLE
restore 'install <alias>' functionality and add better error handling

### PR DESCRIPTION
**Pull request by anurse**
We should now stop on the first error (apparently the outer try...catch does that? I barely know anything about PoSH error handling :)). Also, I trap a few well-known error conditions

Also restored `dnvm install <alias>` as it turned out to be a simple fix.

/cc @NTaylorMullen @glennc @davidfowl

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
Looks good to me but I don't know nearly enough powershell :cry:

File: src/dnvm.ps1 [hunk 1]
```diff
@@ -449,7 +457,12 @@ function Download-Package(
     $wc = New-Object System.Net.WebClient
     Apply-Proxy $wc -Proxy:$Proxy
     _WriteDebug "Downloading $url ..."
-    $wc.DownloadFile($url, $DestinationFile)
+    try {
+        $wc.DownloadFile($url, $DestinationFile)
+    } catch {
+        $Script:ExitCode = $ExitCodes.NoSuchPackage
+        throw "Could not find $runtimeFullName.$Version on feed: $Feed"
```
Comment (davidfowl) [inline src/dnvm.ps1]: Can this null ref? `runtimeFullName`

**Comment by DamianEdwards** [comment]
#165

File: src/dnvm.ps1 (hunk 1 above)
Comment (anurse) [inline src/dnvm.ps1]: It's not a property access :). It's rendering `{runtimeFullName}.{Version}`

**Pull request merged**